src/components/pages/common/PageTitle.js

```diff
@@ -1 +1,31 @@
-// TODO: Write a reusable PageTitle component here using the Contact page's title.
+import React from 'react';
+import { northwesternPurple10 } from 'util/colors';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/styles';
+import Typography from '@material-ui/core/Typography';
+
+export const styles = {
+  page: {
+    margin: 20,
+    padding: 5,
+    borderBottom: `2px solid ${northwesternPurple10}`,
+  },
+};
+
+
+function PageTitle({ classes }) {
```

Comment:
Here, you want to make sure that you are also taking in the `name` prop (which I think should be renamed, see above comments) so that you can actually use it. 

If you run `npm start` (don't forget to run `source .env`, let me know if this part doesn't make sense) and check the pages, you'll notice that the page title actually isn't being rendered. That's because you are trying to display `classes.name` , which doesn't exist. Instead, you want to be displaying `name`.

For a similar example, you can look at `SidebarHeader.js` in `src/components/sidebar`.